openhtf/util/argv.py

```diff
@@ -19,5 +35,7 @@ def __init__(self, *args, **kwargs):
     def __call__(self, parser, namespace, values, option_string=None):
         if hasattr(self, '_proxy'):
             values = self._proxy(parser, namespace, values)
-        setattr(__import__(self._tgt_mod), self._tgt_attr, values)
+        base, mod = self._tgt_mod.rsplit('.', 1)
```

Comment:
This unpacking will break if the target module is already a base module without any dots, but arguably it doesn't make sense to use `StoreInModule` in such a case.